### ISSUE TITLE
cannot download as of 18/03 > yesterday was working fine :(

### ISSUE THREAD

**Issue by durgenheim** [flyingrub/scdl#464]
error = 
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Mission Control\AppData\Local\Programs\Python\Python312\Scripts\scdl.exe\__main__.py", line 7, in <module>
    sys.exit(main())
             ^^^^^^
  File "C:\Users\Mission Control\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\scdl.py", line 158, in main
    if not client.is_client_id_valid():
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Mission Control\AppData\Local\Programs\Python\Python312\Lib\site-packages\soundcloud\soundcloud.py", line 125, in is_client_id_valid
    next(
StopIteration

plz halp

**Comment by AshiraLOL** [comment]
having the same error.
did soundcloud update something new?

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Something\AppData\Local\Programs\Python\Python311\Scripts\scdl.exe\__main__.py", line 7, in <module>
    sys.exit(main())
             ^^^^^^
  File "C:\Users\Something\AppData\Local\Programs\Python\Python311\Lib\site-packages\scdl\scdl.py", line 158, in main
    if not client.is_client_id_valid():
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Something\AppData\Local\Programs\Python\Python311\Lib\site-packages\soundcloud\soundcloud.py", line 125, in is_client_id_valid
    next(
StopIteration

![Screenshot 2024-03-18 232737](https://github.com/flyingrub/scdl/assets/51751845/847f3d50-1286-4a3c-a20b-ff000e668d3f)

**Comment by KeithInvan** [comment]
Linux code blowing the same errors, on the same line numbers with the exception of the first line, is Line 8, not Line 7.

Seems Soundcloud made a change.

**Comment by poliana-girl** [comment]
i'm also getting the same error ); really sad this broke because it was a really useful tool and it doesn't seem to have any active maintainers. i'd try and fix it myself but i would probably make it worse lol.

this works well in the meantime though:
```
yt-dlp --embed-metadata --embed-thumbnail https://soundcloud.com/artist/song
```

**Comment by 7x11x13** [comment]
Should be fixed in v2.7.4

**Comment by gabhab** [comment]
Amazing - thanks so much for fixing it so quickly!
